Tests/MintTests/ProcessInfoExtensionTests.swift

```diff
@@ -0,0 +1,16 @@
+@testable import MintKit
+import XCTest
+
+final class ProcessInfoExtensionTests: XCTestCase {
+    #if arch(x86_64)
+    func testMachineHardwareName_Intel() {
+        XCTAssertEqual(ProcessInfo.processInfo.machineHardwareName, "x86_64")
+    }
+    #endif
+
+    #if arch(arm64)
+    func testMachineHardwareName_AppleSilicone() {
+        XCTAssertEqual(ProcessInfo.processInfo.machineHardwareName, "arm64")
+    }
+    #endif
+}
```

Comment:
Not sure if there is a better way to test this.. While we only need for `machineHardwareName` for macOS, it should also work for Linux as well.

I was going to use `XCTSkipIf(true, "...")` but that's Xcode 11.4+ whereas we need to support Xcode 10.2+.